test/core/promise/activity_test.cc

```diff
@@ -174,7 +174,7 @@ TYPED_TEST(BarrierTest, BarrierPing) {
           return absl::OkStatus();
         });
       },
-      [&scheduler](std::function<void()> f) { scheduler.Schedule(f); },
+      InlineCallbackScheduler(),
```

Comment:
Double checking this is what you want here. The previous version utilized the mock scheduler, and this will now just execute `f` inline.

Reply:
I *think* it was right, but better to be precise here, so I went through and made it use mocks correctly.